README.md

```diff
@@ -17,6 +17,122 @@ Comming soon ;)
 
 ## Configuration
 
+### Basic Configuration
+The basic configuration includes all the minimal configuration to deploy the app successfully. Create a file named `boss.yml` in your project's root directory. The file should include the following:
+
+```yml
+#boss.yml
+project_name: PROJECT_NAME
+project_description: PROJECT_DESCRIPTION
+repository_url: https://github.com/<username>/<project-name>
+user: USERNAME
+port: SERVER_PORT
+```
+
+### Stages
+This includes different instances of your application like `dev`, `qa`, `uat`, `staging`, `production`.
```

Comment:
Stages refer to the configured remote servers where you'd like to deploy your application eg: `dev`, `qa`, `uat`, `staging`, `production`.